docs/en/enterprise-edition/content-collections/runtime-security/agentless-scanning/configure-accounts/configure-gcp.adoc

```diff
@@ -19,44 +19,59 @@ Complete the steps below to configure agentless scanning based on the needs of y
 
 . Go to the *Agentless Scanning* section.
 
-. *Hub account*: Enable the toggle to configure this account as a xref:../agentless-scanning-modes.adoc[hub account].
+. Expand the *Advanced settings* and provide the appropriate values based on your deployment for the following items.
 
-. Expand the *Advanced settings*.
-+
-image::runtime-security/agentless-gcp-configuration.png
+.. xref:#gcp-agentless-modes[Scan settings]
+.. xref:#gcp-agentless-permissions[Scanning permissions]
+.. xref:#gcp-agentless-scope[Scanning scope]
+.. xref:#gcp-agentless-network[Network resources]
```

Comment:
please use the same structure suggested in AWS

Reply:
Done